### PR TITLE
Use strict types instead of docblock and put condition 'if (empty())'…

### PR DESCRIPTION
**Pull request by jhryniuk**
… at the begining of methods.

### CODE REVIEW

**Comment by tarnawski** [comment]
Can one of the admins verify this patch?

**Comment by jhryniuk** [comment]
@tarnawski @partikus @dominikhajduk CR please.

File: src/TaskManager/Storage/InMemory/TaskStorage.php [hunk 1]
```diff
@@ -38,15 +40,10 @@ public function findAll()
             ];
         }
 
-        return empty($tasks) ? [] : $tasks;
```
Comment (tarnawski) [inline src/TaskManager/Storage/InMemory/TaskStorage.php]: My mistake - here should be `isset($tasks) ? [] : $tasks;` but your solution is good too.

Comment (tarnawski) [inline src/TaskManager/Storage/InMemory/TaskStorage.php]: Good job - thanks 👍

**Comment by jhryniuk** [comment]
```
This branch has no conflicts with the base branch
Only those with write access to this repository can merge pull requests.
```

**Pull request merged**